yarn.lock

```diff
@@ -8493,7 +8512,7 @@ postcss-value-parser@^3.0.0, postcss-value-parser@^3.3.0, postcss-value-parser@^
   version "3.3.1"
   resolved "https://registry.yarnpkg.com/postcss-value-parser/-/postcss-value-parser-3.3.1.tgz#9ff822547e2893213cf1c30efa51ac5fd1ba8281"
 
-postcss-values-parser@^2.0.0:
+postcss-values-parser@^2.0.0, postcss-values-parser@^2.0.1:
```

Comment:
All these double minor versions make me think if we should relax `calypso-build` semver just a little bit with `~` (instead of strict or `^`)

Reply:
The way I read this line is that the `^` versions are working as expected, allowing compatible versions of the same package to dedupe to a single version. In this case `^2.0.0` and `^2.0.1` are both satisfied by `2.0.1`. There's no negative impact to longer lines like this.